modules/dnn/perf/perf_net.cpp

```diff
@@ -233,10 +244,8 @@ PERF_TEST_P_(DNNTestNetwork, YOLOv4)
         throw SkipTestException("Test is disabled in OpenVINO 2020.4");
 #endif
     Mat sample = imread(findDataFile("dnn/dog416.png"));
-    cvtColor(sample, sample, COLOR_BGR2RGB);
-    Mat inp;
-    sample.convertTo(inp, CV_32FC3, 1.0f / 255, 0);
-    processNet("dnn/yolov4.weights", "dnn/yolov4.cfg", "", inp);
+    Mat inp = blobFromImage(sample, 1.0 / 255.0, Size(), Scalar(), true);
+    processNet("dnn/yolov4.weights", "dnn/yolov4.cfg", "", {{inp, ""}});
```

Comment:
Just add an override which accepts a single input and make a single element map from it.

Reply:
OK